day_7/high_level.md

```diff
@@ -0,0 +1,8 @@
+##### Checker Board Notes
+
+1. The first thing I had to do was collect the dimensions of the board from the user. In order to do this I was able to use a simple gets.chomp code and assign the number to a variable that way I could use it later on. I also decided to convert the provided number to an integer to ensure I wouldn’t have issues later on.  
+2. The next step was to figure out the “psuedocode” for the program in order to help me have a direction towards figuring out the code. I knew that after I was given my variable, I would need to run through a series of actions a certain number of times, so I looked up loops. I decided to use the “while” loop because I didn’t need to know the number of times I would go through the loop beforehand, meaning I could plug any number in and it would work.  
+3. While I was learning how “while” loops worked, I knew that I would need my variable to run through a series of “if-statements” so I could control what character was printed and in what order. It took me a while to understand the logic for these conditionals in combination with “while” loops, I did a lot of google searching and asked for help from some of my peers as much as possible. Originally I was thinking that I would only need two conditionals, one for “X” and one for “ “ but I eventually realized that I actually needed three conditionals in order to see the results I was looking for.   
+4. I then found it was best to put all of those actions into a “defined method” that way I could easily call on it later and I wouldn’t need to repeat my code as much.   
```

Comment:
Nice work overall on breaking down checkerboard. I especially like this line where you distill the practical purpose of defining methods.